postgres/postgresql.conf.erb

```diff
@@ -33,13 +32,20 @@ lc_numeric = 'en_AU.UTF-8'
 lc_time = 'en_AU.UTF-8'
 default_text_search_config = 'pg_catalog.english'
 
+idle_in_transaction_session_timeout = '120s'
+
 # These settings only apply to the master.
-wal_level = hot_standby
+wal_level = replica
```

Comment:
I just read up on it, indeed `logical` does trigger different behaviour. It's unclear how much the WAL volume will increase by, I guess we could try it out and see. It definitely sounds like logical replication is the way to go for flexibility and future compatibility (it also sounds like it's likely to be rolled into postgres 10 core).